Modules/Relationships/src/Relationships.Infrastructure.Database.Postgres/Relationships.Infrastructure.Database.Postgres.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+	<BaseOutputPath>..\Relationships.Infrastructure\bin\</BaseOutputPath>
```

Comment:
This line doesn't exist in the other projects. So there are 2 possibilities:
1. You didn't run the application with a Postgres database
2. The BaseOutputPath is not necessary after all.

I hope it's the option 2. :-)

Reply:
It's option 2!